docassemble/tclpgoogledocsmerger/data/questions/search_and_download.yml

```diff
@@ -92,8 +92,8 @@ question: |
   No matching clauses found.
   % endif
 subquestion: |
-  ${ filled_template['final'] }
-  
+  ${ filled_template['final'].pdf }
```

Comment:
I guess this is intentional, but did we want users to download the combined forms as a PDF or as a DOCX, or to give them the choice?

Reply:
DOCX ultimately--I did this for testing but we can remove it for now.

Reply:
```suggestion
  ${ filled_template['final'] }
```